frontend/pages/milestones/MilestoneListPage.jsx

```diff
@@ -1,5 +1,145 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import Mytable from '@components/common/table';
+// 마일스톤 리스트 테이블
+import service from '@services';
+// 마일스톤 데이터
+import styled from 'styled-components';
+// import MainPageLayout from '@layouts/MainPageLayout';
+import { GoMilestone, GoCheck, GoCalendar } from 'react-icons/go';
 
-const MilestoneListPage = () => <div />;
+const MilestoneListPage = () => {
+  const [open, setOpenMilestone] = useState([]);
+  const [close, setClosedMilestone] = useState([]);
+  const [data, setData] = useState([]);
+  const clickEditBtn = async () => {
+    // 수정버튼
+  };
+  const clickCloseBtn = async () => {
+    // close버튼
+  };
+  const clickDeleteBtn = async () => {
+    // dekete 버튼
+  };
+  const getOpen = async () => {
+    const milestones = await service.getOpenMilestones();
+    setOpenMilestone(milestones.data);
+  };
+  const getClosed = async () => {
+    const milestones = await service.getClosedMilestones();
+    setClosedMilestone(milestones.data);
+  };
+  const getData = async (status) => {
+    const milestones =
+      status === 'close' ? await service.getClosedMilestones() : await service.getOpenMilestones();
+    setData(milestones.data);
+  };
+  useEffect(() => {
+    getOpen();
+    getClosed();
+    getData();
```

Comment:
getDate에 status 인자를 넣어주는게 더 명확할 것 같습니다
```suggestion
    getData('close');
```